src/main/java/org/breedinginsight/brapi/v2/ListController.java

```diff
@@ -80,7 +80,7 @@ public HttpResponse<Response<DataResponse<Object>>> getLists(
             // get germplasm lists by default
             BrAPIListTypes type = BrAPIListTypes.fromValue(queryParams.getListType());
             if (type == null) {
-                type = BrAPIListTypes.GERMPLASM;
```

Comment:
I think that this goes against the BrAPI spec.  I think we should probably return all lists if the `type` parameter is `null`.  Worth confirming with @BrapiCoordinatorSelby

Reply:
The BrAPI Spec does not specifically define what should happen if `listType` is null. 
I would expect it to return all lists of all types, in line with other GET query parameters. You might also make `listType` a required parameter for your implementation. 
Defaulting `listType` to germplasm is unusual, and might cause some confusion if it is not well documented, but it is not technically against the BrAPI spec.

Reply:
Thanks for the input @BrapiCoordinatorSelby, all lists of all types are now returned.